com.discordapp.DiscordCanary.json

```diff
@@ -1,11 +1,11 @@
 {
     "app-id": "com.discordapp.DiscordCanary",
     "base": "org.electronjs.Electron2.BaseApp",
-    "base-version": "22.08",
+    "base-version": "23.08",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "22.08",
+    "runtime-version": "23.08",
     "sdk": "org.freedesktop.Sdk",
-    "command": "discord-canary",
+    "command": "com.discordapp.DiscordCanary",
```

Comment:
I don't see a reason to change the executable name? It doesn't need to be strictly rDNS

Reply:
Mirrored from https://github.com/flathub/com.discordapp.Discord/commit/73ec9ac133143ebbabdf4287e0f77cbe09d3a412.
I'm not a KDE user, so I can't check if the issue is still relevant, but I've reflected these changes just in case

Reply:
Hm ok. It's fine then